client/index.jsx

```diff
@@ -1,8 +1,51 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import App from './app';
+import React, { useRef } from 'react';
```

Comment:
I'm trying to update our code base to stop using `index.js` for anything except getting the container populated. Could you move the MusicWidget to a separate file? Thanks.